oncoact/patientreporter/reporting_pipeline/share_report.py

```diff
@@ -51,12 +44,15 @@ def share_report(self, publish_to_portal, notify_users):
         self._delete_old_artifacts_in_portal_bucket()
         self._delete_old_artifacts_in_gcp_share_bucket()
 
-        self._copy_files_to_remote_buckets(publish_to_portal=publish_to_portal)
+        publish_to_portal_and_notify = self.rest_client.get_lama_patient_reporter_data(self.report_created_record['barcode'])[
```

Comment:
Nitpick: maybe rename to `is_shared_through_portal`, I feel like this reads quite nicely:

```python
        response = self.rest_client.post_report_shared(report_created_id=self.report_created_record['id'], publish_to_portal=is_shared_through_portal, notify_users=is_shared_through_portal)
```